docs/developerguide.rst

```diff
@@ -51,3 +51,422 @@ code directly editable within pods, you can use ``reana-cluster-dev.yaml``.
+           name: reana-workflow-monitor-code
+           path: "/code/reana-workflow-monitor:/code"
+       environment:
+         - ZMQ_PROXY_CONNECT: tcp://zeromq-msg-proxy.default.svc.cluster.local:8667
+         - WDB_SOCKET_SERVER: "wdb"
+         - WDB_NO_BROWSER_AUTO_OPEN: "True"
+         - FLASK_DEBUG: "1"
+
+     reana-workflow-engine-yadage:
+       type: "docker"
+       image: "reanahub/reana-workflow-engine-yadage:0.1.0"
+       mountpoints:
+         - type: hostPath
+           name: reana-workflow-engine-yadage-code
+           path: "/code/reana-workflow-engine-yadage:/code"
+       environment:
+         - ZMQ_PROXY_CONNECT: "tcp://zeromq-msg-proxy.default.svc.cluster.local:8666"
+         - WDB_SOCKET_SERVER: "wdb"
+         - WDB_NO_BROWSER_AUTO_OPEN: "True"
+
+.. code-block:: console
+
+   #development-kubeconfig.yaml
+   apiVersion: v1
+   clusters:
+   - cluster:
+       # Since minikube generates self-signed certificate that doesn't include
+       # hostname `localhost` TLS hostname verification has to be skipped.
+       insecure-skip-tls-verify: true
+       server: https://localhost:8443
+     name: minikube
+   contexts:
+   - context:
+       cluster: minikube
+       user: minikube
+     name: minikube
+   current-context: minikube
+   kind: Config
+   preferences: {}
+   users:
+   - name: minikube
+     user:
+       as-user-extra: {}
+       token: $TOKEN
+
+Note that you must change the value of $TOKEN to the token you acquired in step 2.
+
+6. Instruct ``reana-cluster`` to use your own reana-cluster.yaml by using ``-f`` flag:
+
+.. code-block:: console
+
+   $ reana-cluster -f $(pwd)/reana-cluster-dev.yaml verify backend
+
+7. Configure ``kubectl`` to connect to K8S API on the remote host running minikube
+
+Kubectl supports defining configuration by supplying path to kubeconfig
+configuration file by ``$KUBECONFIG`` environment variable.
+(https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable)
+
+.. code-block:: console
+
+   $ export KUBECONFIG=$(pwd)/development-kubeconfig.yaml
+   $ kubectl cluster-info
+   > Kubernetes master is running at https://localhost:8443
+
+You should now be able interact with Kubernetes API of your Minikube VM on
+remote host with both ``reana-cluster`` and ``kubectl``.
+
+Expose Docker daemon on the remote host running Minikube
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+**AT REMOTE HOST:**
+
+Run alpine/socat docker container that maps your docker.sock to tcp port 2375.
+Note that docker.sock is exposed as plain HTTP without authentication, so
+don't expose it outside 127.0.0.1 of remote host running minikube.
+SSH port forwarding is used to provide a secure connection to port.
+
+1. Share docker.sock by HTTP at port 2375
+
+   If Minikube has been started with ``--vm-driver=none`` run following command.
+
+.. code-block:: console
+
+   $ docker run -d --restart=always \
```

Comment:
You can indent a bit the 2nd and 3rd line to have spaces there, for clarity:

```
$ first line \
    second line \
    another continuation \
    ...
```

Reply:
fixed